universum/modules/launcher.py

```diff
@@ -289,7 +289,7 @@ def _handle_postponed_out(self):
 
 @needs_output
 @needs_structure
-class Launcher(ProjectDirectory):
+class Launcher(ProjectDirectory, ErrorState):
```

Comment:
codestyle/convention question: should ErrorState be inherited first or last?

Reply:
In python it only affects which members are shadowed.
If two bases are totally unrelated, the order doesn't matter.
I usually add a new base to the end of the list.
At least, this way I don't need to think about it too much.